.github/workflows/release.yml

```diff
@@ -5,19 +5,197 @@ on:
   workflow_dispatch:
 
 jobs:
+  create-release:
+    name: Create Release
+    runs-on: ubuntu-20.04
+    outputs:
+      mobile-version: ${{ steps.retrieve-mobile-version.outputs.mobile_version }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f  # v2.3.4
+        with:
+          ref: rc
+
+      - name: Retrieve Mobile release version
+        id: retrieve-mobile-version
+        run: |
+          ver=$(sed -n -e '/android:versionName/ s/.*\= *//p' ./src/Android/Properties/AndroidManifest.xml | tr -d '"')
+          echo "::set-output name=mobile_version::${ver}"
+        shell: bash
+
+      - name: Check to make sure Mobile release version has been bumped
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          latest_ver=$(hub release -L 1 -f '%T')
+          latest_ver=${latest_ver:1}
+          echo "Latest version: $latest_ver"
+          ver=${{ steps.retrieve-mobile-version.outputs.mobile_version }}
+          echo "Version: $ver"
+          if [ "$latest_ver" = "$ver" ]; then
+            echo "Version has not been bumped!"
+            exit 1
+          fi
+        shell: bash
+
+      - name: Create Draft Release
+        uses: actions/create-release@0cb9c9b65d5d1901c1f53e5e66eaf4afd303e70e  # v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: v${{ steps.retrieve-mobile-version.outputs.mobile_version }}
+          release_name: Version ${{ steps.retrieve-mobile-version.outputs.mobile_version }}
+          draft: true
+          prerelease: false
```

Comment:
I might be remembering this incorrectly (or interpreted it incorrectly to begin with), I was under the impression we were moving away from the draft release and moving towards doing all of the deploying/releasing to stores and then end with a release job that gathers all build artifacts/release assets and creates a published release? Or do we want to keep the draft release?

Reply:
What are your thoughts on moving the create release to the end of the workflow and switching to something like this?

```
- name: Create release
  uses: ncipollo/release-action@95215a3cb6e6a1908b3c44e00b4fdb15548b1e09
  with:
    artifacts: release-asset.zip
    commit: ${{ github.sha }}
    tag: "${{ needs.version.outputs.tag_version }}"
    name: "Version ${{ needs.version.outputs.release_version }}"
    body: "<insert release notes here>"
    token: ${{ secrets.GITHUB_TOKEN }}
    draft: true
```

Reply:
I like that action.  I've updated the code to use it.